types/slimerjs/index.d.ts

```diff
@@ -147,7 +147,7 @@ interface IWebPage {
     render(filename: string): void;
     renderBase64(format: string): string;
     sendEvent(mouseEventType: string, mouseX?: number, mouseY?: number, button?: string): Promise<void>;
-    sendEvent(keyboardEventType: string, key: string, null1?: void, null2?: void, modifier?: number): Promise<void>;
+    sendEvent(keyboardEventType: string, key: string, null1?: undefined, null2?: undefined, modifier?: number): Promise<void>;
```

Comment:
I changed it to null, maybe you can help

here is the api docu
 
<img width="739" alt="bildschirmfoto 2017-03-28 um 22 36 02" src="https://cloud.githubusercontent.com/assets/316469/24426377/cfd61d38-1407-11e7-8339-fa619132f86c.png">